examples/8-eks-cluster-with-eks-addons/main.tf

```diff
@@ -64,6 +64,7 @@ locals {
   vpc_cidr       = "10.0.0.0/16"
   vpc_name       = join("-", [local.tenant, local.environment, local.zone, "vpc"])
   eks_cluster_id = join("-", [local.tenant, local.environment, local.zone, "eks"])
+  cluster_name   = join("-", [local.tenant, local.environment, local.zone, "eks"])
```

Comment:
Good Spot but this needs to be replaced with `eks_cluster_id`. I will update the example directly in `examples-migration` branch.

You can close this PR